Custom config file (#98)
mirumee/ariadne-codegen: README.md

## Configuration

> `ariadne-codegen` reads configuration from `[tool.ariadne-codegen]` section in your `pyproject.toml`'. You can change file, configuration is read from, using `--config` option, eg. `ariadne-codegen --config custom_file.toml`

Reviewer: ```suggestion
`ariadne-codegen` reads configuration from `[tool.ariadne-codegen]` section in your `pyproject.toml`'. You can use other configuration file with `--config` option, eg. `ariadne-codegen --config custom_file.toml`
```
